test/fixtures/comments_with_ci.json

```diff
@@ -25,5 +25,8 @@
   },{
     "publishedAt": "2018-02-09T21:38:30Z",
     "bodyText": "CI https://ci.nodejs.org/job/node-test-commit-lite/246/"
+  },{
+    "publishedAt": "2017-10-21T04:16:36.458Z",
+    "bodyText": "CI https://ci.nodejs.org/job/node-test-pull-request-lite-pipeline/7213/pipeline/"
```

Comment:
Can you use something similar to what the bot actually posts? It usually goes like:

```
@Someone build started: https://ci.nodejs.org/blue/organizations/jenkins/node-test-pull-request-lite-pipeline/detail/node-test-pull-request-lite-pipeline/493/pipeline
```

Reply:
Updated, to use syntax in form that bot posts.